pkg/assets/config/test-keeper.yaml

```diff
@@ -99,4 +99,8 @@ skip_validation_for:
 
   # Shell
   - '*.sh'
-  - '*.bat'
\ No newline at end of file
+  - '*.bat'
+
+  # Our own configurations
+  - 'test-keeper.yaml'
+  - 'test-keeper.yml'
```

Comment:
@MatousJobanek this probably is a very strong argument to move all our internal configuration, hints et al to single location like `.ike-prow/` - would make it one line ;) Issue coming.

Reply:
I can only agree with you, so +1 from my side